tests/atoms/gaia_apps.js

```diff
@@ -372,12 +372,31 @@ var GaiaApps = {
     };
     return result;
   },
+  /**
+  * Install the app with the specified ManifestURL
+  */
+  installWithManifestURL: function(manifestURL) {
```

Comment:
Hence, I wonder if giving this precision in the function name is worth it. How about some simpler as install and uninstall? 

=> you mean instead of installWithManifestURL and uninstallWithManifestURL ?